packages/react-components/src/components/timegraph-output-component.tsx

```diff
@@ -337,7 +337,7 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
                     height: parseInt(this.props.style.height.toString()),
                     width: this.props.style.chartWidth, // this.props.style.mainWidth,
                     backgroundColor: this.props.style.chartBackgroundColor,
-                    lineColor: this.props.style.lineColor,
+                    lineColor: this.props.backgroundTheme === 'light' ? 0xdddddd : 0x34383C,
```

Comment:
I think we should keep the original code here and update the TimeAxisComponent instead to have the previous color.